google/cloud/logging_v2/entries.py

```diff
@@ -46,11 +46,12 @@
 )
 
 
-def logger_name_from_path(path):
+def logger_name_from_path(path, project=None):
```

Comment:
nit: Perhaps I am not well familiar with Python, but is there a way to create another `logger_name_from_path` function with `project` parameter? I think that it would be better then having parameters with default values

Reply:
I think this is the Pythonic way to do it.  Especially since this function is just directly calling another function. If it wasn't a public function already in use, I would prefer to remove this one altogether

What's the argument against default parameters?